### ISSUE TITLE
Hard to resize overlay

### ISSUE THREAD

**Issue by BDubbB** [extesy/DeckTracker#110]
Most of the time the overlay won't resize when I click and drag on, or even near, the borders. Most times it either just moves the overlay or does nothing. This is happening in Elder Scrolls Legends, by the way.

**Comment by Timshady79** [comment]
this.

**Comment by extesy** [comment]
@Timshady79 you can manually delete or edit config file, see second to last item  it https://github.com/extesy/DeckTracker#faq-frequently-asked-questions list

**Comment by oraculus** [comment]
yes please, right now is really hard to control the size, specially if I want to shrink it. I all ways end unwillingly increasing it instead.

**Comment by dasher3** [comment]
Thanks for the tip about config file. 

Even so, it really needs to be resizable in game, I thought it was the lower right had corner, but today nothing worked.

**Comment by BDubbB** [comment]
@dasher3 

> I thought it was the lower right had corner, but today nothing worked.

The zone that seems to work best for resizing is just a pixel or two to the left of the right hand vertical line a short distance up from the corner.

In my opinion, it would be nice if the border were able to be seen easily, clicking and dragging from absolutely any part of the border resized it along the appropriate axis, the border in the corner resized along both axes, the overlay not resized by clicking and dragging outside the border (e.g., on the play field), and the resizing never stopped while the mouse button was held down after a click and drag on the border. I think that those things would greatly enhance the usability of this tracker (which is a pretty great tracker already).

**Comment by BDubbB** [comment]
This is a really important issue! The video that is linked to showing how to install and use the deck tracker, https://www.youtube.com/watch?v=-iHewjQG1S0, shows how frustrating it can be trying to resize the thing (you can see it at 7:30, 9:32, and other timestamps).

@extesy 
Is there any chance that we can get a fix to this soon? I think that this might be the only thing holding the deck tracker back from being super popular. I still love it and appreciate it, however.

**Comment by extesy** [comment]
The thing is - I've also seen opposite case in somebody's stream when they accidentally click on a resizer edge and change overlay size unintentionally. It's a balance act and if I increase the active zone it start annoying people with low resolution screens. I'm not sure what would be a right solution here. Maybe make it depend on the screen resolution?

**Comment by BDubbB** [comment]
@extesy 
Yeah, I don't think that the issue is with the size of the resizer edge. However, I can think of a couple things that would help. (1) If there were a visual indicator of the resizer zone, then it would be easier to tell where to click when you begin your click and drag. (2) If the resize, after the click and drag, happened reliably. It is annoying that while you are still holding the mouse down, the overlay stops resizing and that you can't drag back and forth (it seems like you can only increase or decrease with a single click and drag) (3) Finally, it seems that clicks and drags on the play field should not resize the overlay when the click occurred outside of the resizer zone.

I haven't done any programming in over a decade, but if you want me to just check and see if there is something that I can do, then I would be willing to have a look. I was never any good at programming, though, so I could probably only make suggestions.

**Comment by BDubbB** [comment]
@extesy 

I just ran into a problem where, during the final episode of Return to Clockwork city, I tried playing Dragon Aspect and this resulted in the overlay being resized to maximum. This is why it should be impossible to resize if the click and drag was not initiated from within the border, or even just the resizer area, of the overlay.

This is the result: 

![untitled](https://user-images.githubusercontent.com/10868982/33644701-60b81a54-d9fb-11e7-8317-e46b77fadaf9.jpg)

I had to disable the overlay with F2 in order to finish the match. When I proceeded to the next story match, and re-enabled the overlay, this is what it looked like:

![untitled2](https://user-images.githubusercontent.com/10868982/33644739-938f8282-d9fb-11e7-8ecd-e85dfd259aa3.jpg)

Resizing desperately needs attention.

It would also be very helpful, especially in cases like this, if there were a hotkey, probably an F-key (in keeping with the current hotkeys, which would reinitialize only the overlay size. Once again, please don't take this as criticism. I LOVE this your deck tracker, and I greatly appreciate your work in bringing it to the community and making it better.

**Comment by extesy** [comment]
Hm... This stuff shouldn't happen, I've explicitly added a protection against overlays being larger than a screen size. Not sure why it wasn't working in your case. I'll take a look. In the meantime, you can delete config file which would reset overlay size.

**Comment by extesy** [comment]
Deck size getting too large should be fixed in release 1.0.73
